src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -4,4 +4,7 @@
 
 public interface RegistrationService {
     User register(User user);
+
+    boolean isSuccesfullyRegistered(User user);
```

Comment:
The method 'isSuccesfullyRegistered' is not described in the task requirements. If it's not required for the project's functionality, it should be removed to adhere to the task guidelines.